tezos/messages/src/p2p/encoding/advertise.rs

```diff
@@ -35,5 +36,11 @@ impl AdvertiseMessage {
 
 cached_data!(AdvertiseMessage, body);
 has_encoding!(AdvertiseMessage, ADVERTISE_MESSAGE_ENCODING, {
-    Encoding::Obj(vec![Field::new("id", Encoding::list(Encoding::String))])
+    Encoding::Obj(vec![Field::new(
+        "id",
+        Encoding::blist(
```

Comment:
@akoptelov 
I would prefer full name `blist` -> `bounded_list`,
ocaml code is full of one-letter variables and it is pain to read it sometimes

Reply:
OK.